compiler/zrc_typeck/src/typeck/scope.rs

```diff
@@ -156,10 +156,13 @@ pub struct ValueCtx<'input> {
     /// sibling scopes.
     mappings: HashMap<&'input str, Rc<RefCell<ValueEntry<'input>>>>,
 }
+impl Default for ValueCtx<'_> {
```

Comment:
The comment on line 161 is clear as to why ValueCtx does not impl Default. If a Clippy lint was causing you trouble, you can `#[expect(...)]` it on the struct.